File-Formats/PDF/Working-with-Annotations.md

```diff
@@ -1507,6 +1507,269 @@ document.Close(True)
+
+lPage.Annotations.Add(richMediaAnnotation);
+
+//Save the document to the disk.
+
+loadedDocument.Save(@"RichMediaAnnotation.pdf");
+
+loadedDocument.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+
+'Load the existing PDF document.
+
+ Dim loadedDocument As New PdfLoadedDocument("input.pdf")
+
+'Get the page. 
+
+Dim lpage As PdfLoadedPage = TryCast(loadedDocument.Pages(0),PdfLoadedPage)
+
+'Create the PDF richmedia annotation.
+
+Dim richMediaAnnotation As New PdfRichMediaAnnotation (new RectangleF(0,0,200,100)) 
+
+richMediaAnnotation.ActivationMode = PdfRichMediaActivationMode.Click
+
+richMediaAnnotation.PresentationStyle = PdfRichMediaPresentationStyle.Windowed
+
+Dim content As New PdfRichMediaContent (@"video.mp4")
+
+richMediaAnnotation.Content = content
+
+'Create the appearance of the richmedia.
+
+richMediaAnnotation.Appearance.Normal.Graphics.DrawString("Click here to play video...",new PdfStandardFont(PdfFontFamily.Helvetica,15),PdfBrushes.Blue,new RectangleF(0,0,200,100),new PdfStringFormat(PdfTextAlignment.Center,PdfVerticalAlignment.Middle))
+
+'Add the annotation to the page. 
+
+lpage.Annotations.Add(richMediaAnnotation)
+
+'Save the document to the disk. 
+
+loadedDocument.Save("RichMediaAnnotation.pdf") 
+
+loadedDocument.Close(True)
+
+{% endhighlight %}
+
+  {% highlight c# tabtitle="UWP" %}
+//Load the existing PDF document.
+
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument("input.pdf");
+
+//Get the page.
+
+PdfLoadedPage lpage = loadedDocument.Pages[0] as PdfLoadedPage;
+
+//Create the PDF richmedia annotation.
+
+PdfRichMediaAnnotation richMediaAnnotation = new PdfRichMediaAnnotation(new RectangleF(0,0,200,100));
+
+//Set properties to the annotation.
+
+richMediaAnnotation.ActivationMode = PdfRichMediaActivationMode.Click;
+
+richMediaAnnotation.PresentationStyle = PdfRichMediaPresentationStyle.Windowed;
+
+//Set the richmedia content.
+
+PdfRichMediaContent content = new PdfRichMediaContent(@"video.mp4");
+
+richMediaAnnotation.Content = content;
+
+//Create the appearance of the richmedia. 
+
+richMediaAnnotation.Appearance.Normal.Graphics.DrawString("Click here to play video...",new PdfStandardFont(PdfFontFamily.Helvetica,15),PdfBrushes.Blue,new RectangleF(0,0,200,100),new PdfStringFormat(PdfTextAlignment.Center,PdfVerticalAlignment.Middle)); 
+
+//Add the annotation to the page. 
+
+lpage.Annotations.Add(richMediaAnnotation);
+
+//Save a PDF document to the stream.
+
+MemoryStream stream = new MemoryStream(); 
+
+await loadedDocument.SaveAsync(stream);
+
+//Close the document. 
+
+loadedDocument.Close(true);
+
+//Save the stream as a PDF document file in the local machine. Refer to the PDF/UWP section for the respective code samples. 
+
+Save(stream, "RichMediaAnnotation.pdf");
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+
+//Load the existing PDF document.
+
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument("input.pdf");
```

Comment:
We don't have support to directly load the input document using the file. Check our existing UG content and fix this.